audio_video_recorder/README.md

```diff
@@ -0,0 +1,78 @@
+# audio_video_recorder
+
+ROS package for recording audio and video synchronously
+
+![[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+](.media/pr2_sample.gif)
+
+[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+
+## Sample
+
+You can record audio and video on your laptop.
```

Comment:
please describe how do you record both audio and video on your computer
`audio and video as avi file? or both avi and mp3 file? on your computer`

laptop? maybe your computer might be better.

Reply:
We record audio and video in one AVI file.
I will update the readme.